test-lib.sh

```diff
@@ -1147,9 +1148,11 @@ ct_check_latest_imagestreams() {
     [ -f "$latest_version/.exclude-$OS" ] && latest_version=$(grep '^VERSIONS' Makefile | rev | cut -d ' ' -f 2 | rev )
     # Only test the imagestream once, when the version matches
     # ignore the SC warning, $VERSION is always available
+
+    test_lib_dir=$(dirname "$(readlink -f "$0")")
+    python3 "${test_lib_dir}/show_all_imagestreams.py"
```

Comment:
Do we want to test something with calling `show_all_imagestreams.py`, or is it here for debugging purposes?

Reply:
Test is present here: https://github.com/sclorg/container-common-scripts/pull/372/files#diff-9c2d3c45e3a4d98424608dc9f1aad1ed04f13cd5d79c3364c6d2a9dd99548dd6R13

Reply:
The output is only for information.